src/Task/Base/ParallelExec.php

```diff
@@ -127,6 +146,9 @@ public function run()
             $process->setTimeout($this->timeout);
             $process->start();
             $this->printTaskInfo($process->getCommandLine());
+            if ($this->waitInterval) {
+                sleep($this->waitInterval);
```

Comment:
Bonus points pt. 2: encapsulate the enqueue / dequeue / start process into a separate class.